app/assets/stylesheets/themes/default/pages/_matches.scss

```diff
@@ -210,16 +191,9 @@ div#match {
       @include span-columns(6);
       @include omega;
 
-      .controls {
-        right: 0;
-      }
-
-      form {
-
-        label.checkbox {
-          float: left;
-          margin-right: 1em;
-        }
+      label.checkbox {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 4